db/migrate/20200805150707_remove_github_domain_column_from_application.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveGithubDomainColumnFromApplication < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
I'd have thought this could be a `change` method and you'd not need the down one.

Reply:
Done!